code/game/area/areas.dm

```diff
@@ -20,6 +20,8 @@
 	var/blob_allowed = TRUE // Does it count for blobs score? By default, all areas count.
 	var/clockwork_warp_allowed = TRUE // Can servants warp into this area from Reebe?
 	var/clockwork_warp_fail = "The structure there is too dense for warping to pierce. (This is normal in high-security areas.)"
+	var/abductor_warp_allowed = TRUE // Similar to above, for abductors! Mostly to protect the AI.
```

Comment:
Better move is to have a blacklist of area types stored somewhere related to abductors.

Reply:
Can be done, I'll get around to it.